test/atomics.wast

```diff
@@ -1,7 +1,7 @@
 (module
  (type $0 (func))
  (memory $0 23 256 shared)
- (func $atomics (type $0)
+ (func $atomic-loadstoare (type $0)
```

Comment:
This is the original handwritten test file, so it was just an old-fashioned typo in my fingers :)